<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLoading</class>
 <widget class="QDialog" name="DialogLoading">
  <property name="minimumSize">
   <size>
    <width>394</width>
    <height>60</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>394</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/img/ic_folder.png</normaloff>:/img/ic_folder.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelProg">
       <property name="minimumSize">
        <size>
         <width>29</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>29</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>100%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelCurrentFile">
     <property name="text">
      <string>Copying : </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
